rendezvous.php

```diff
@@ -110,6 +111,7 @@ function book_form2($ren_ses_id)
     {
       $looking_for_free_slot = true;
       echo '<form name="book_form2" method="POST" action="">';
+      csrfToken();
```

Comment:
Again, you have the same problem here that this print will invalidate all other tokens in this page. With the current implementation `csrfToken()` can only be called **once** per page and you need to make all calls **after** all calls to the validation function.